S/StaticArrays/Compat.toml

```diff
@@ -16,5 +16,5 @@ Compat = "0.46-2"
 ["0.7.1-0.7"]
 Compat = "0.66-2"
 
-["0.8-0"]
+["0.8-0.10"]
 julia = "0.7-1"
```

Comment:
Please merge https://github.com/JuliaArrays/StaticArrays.jl/pull/617 and retrigger the bot in order to keep this bound.